engine/src/main/java/org/hibernate/search/engine/metadata/impl/AnnotationMetadataProvider.java

```diff
@@ -1040,7 +1040,9 @@ private void initializeMemberLevelAnnotations(String prefix,
 		);
 		checkForContainedIn( member, typeMetadataBuilder, parseContext );
 
-		numericFields.validate();
+		if ( !parseContext.skipFieldBridges() ) {
```

Comment:
Puzzling check. I guess I could find out the reasons by debugging the tests, but this means some comments here could be helpful?